llama_index/vector_stores/types.py

```diff
@@ -158,6 +158,10 @@ def add(
         """Add embedding results to vector store."""
         ...
 
+    def get_by_id(self, doc_id: str) -> List[str]:
```

Comment:
I don't think we want to add this to the base class just yet -- I think there will be many errors about this not being implemented by the other classes